Verduidelijk “Over NL Design System” (#726)
nl-design-system/documentatie: docs/project/over-nlds.mdx

Overheidsorganisaties hebben veel overlap in de online diensten die ze aanbieden. Die overlap maakt samenwerken logisch. Want als we dezelfde uitdagingen hebben, kunnen we vast ook van elkaar leren wat betreft oplossingen. Van formulieren (eenvoudig of complex) tot zaaksystemen. Van digitale processen tot complexe kaarten. Al dit soort websites en applicaties moeten toegankelijk, inclusief en gebruiksvriendelijk zijn.

Enkele voordelen van samenwerken:

- **Voor specialisten** (zoals front-end, ontwerp): je kunt bestaande bouwstenen met vertrouwen overnemen en/of meewerken aan nieuwe.
- **Voor leveranciers**: je kunt overheidsklanten makkelijker en consistenter bedienen.
- **Voor managers en beslissers**: je teams werken met bouwstenen die ook bij andere overheidsorganisaties worden bewezen.

## De NL Design System aanpak

### Onderdelen

NL Design System bevat verschillende soorten onderdelen:

- [richtlijnen](/richtlijnen): adviezen over verschillende onderwerpen, zoals het [maken van formulieren](/richtlijnen/formulieren/).
- [componenten](/componenten): bouwblokken waarmee websites en applicaties kunnen worden opgebouwd, zoals knoppen, invulvelden en tabellen.

> - [patronen](/voorbeelden/patronen): voorbeelden van hoe componenten samen kunnen worden gebruikt voor specifieke taken die gebruikers willen doen, op basis van gebruikersonderzoeken en inzichten.

Reviewer: ```suggestion
- [patronen](/voorbeelden/patronen): voorbeelden van hoe componenten samen kunnen worden gebruikt voor specifieke taken die gebruikers willen doen, op basis van inzichten uit [gebruikersonderzoek](voorbeelden/onderzoek).
```

### Hoe we werken

Deze onderdelen ontstaan niet bij het kernteam, maar in de community. Bijvoorbeeld omdat een gemeente of ministerie een specifiek component nodig heeft. Als blijkt dat meer organisaties datzelfde componenten willen gebruiken, wordt er samengewerkt aan een ultieme versie, waarop het kernteam garanties geeft.

> Deze aanpak om samen te werken noemen we het “[estafettemodel](/handboek/estafettemodel)”. Hierin krijgen richtlijnen, componenten en patronen een status, die aangeeft hoe ver ze zijn gevorderd. Zo gaan onderdelen van “Help Wanted” naar “Community” naar “Candidate” naar “Hall of Fame” (zie: [uitleg per status](/handboek/estafettemodel)). Elke stap heeft een eigen checklist, waarmee we kwaliteit waarborgen en bepalen wie wat doet.

Reviewer: ```suggestion
Deze aanpak om samen te werken noemen we het “[estafettemodel](/handboek/estafettemodel)”. Hierin krijgen richtlijnen, componenten en patronen een status, die aangeeft hoe ver ze zijn gevorderd. Zo gaan onderdelen van [“Help Wanted”](/handboek/estafettemodel#help-wanted) naar [“Community”](/handboek/estafettemodel#community) naar [“Candidate”](/handboek/estafettemodel#candidate) naar [“Hall of Fame”](/handboek/estafettemodel#hall-of-fame). Elke stap heeft een eigen checklist, waarmee we kwaliteit waarborgen en bepalen wie wat doet.
```

- onze componenten kunnen worden gebruikt om toegankelijke producten te maken. Op “Hall of Fame” niveau garanderen we zelfs toegankelijkheid en zijn de componenten in productie getest.
- onze richtlijnen kunnen worden gebruikt als kennisbank, ze bevatten concrete adviezen over wat er nodig is en waarom. Er zijn do's en don'ts, uitleg van WCAG criteria en tips die verder dan WCAG gaan.

## Leveranciers

Veel organisaties werken met NL Design System via hun leveranciers. Bijvoorbeeld doordat hun leveranciers onze componenten gebruiken, of meedoen in de community.

Ben je zelf leverancier en wil je voor klanten met NL Design System aan de slag? Neem dan contact op met het [kernteam](/project/kernteam), we helpen je graag op weg.

## Wie zijn jullie?

### Kernteam en community

We zijn een samenwerking van specialisten, van gemeenten tot Rijksoverheid. Het [kernteam](/project/kernteam) faciliteert deze samenwerking. Specialisten zijn bijvoorbeeld ontwerpers, ontwikkelaars en onderzoekers. Ze werken in hun organisatie aan specifieke webprojecten en delen hun ervaring in de community.

#### Wat doet de community?

> Specialisten in de community denken bijvoorbeeld samen na over hoe een Mijn Zaken-systeem het beste kan worden ingericht, hoe tussenruimte in vormgeving het beste kan worden opgelost of hoe je een toegankelijk formulier bouwt. De community bouwt ook de daadwerkelijke componenten en test ze in hun eigen websites en applicaties.

Reviewer 2: Ik denk dat dit 2e voorbeeld niet zal worden begrepen door iedereen. Daarom weglaten?

```suggestion
Specialisten in de community denken bijvoorbeeld samen na over de beste inrichting van een Mijn Zaken-systeem of een toegankelijk meerstappen formulier. De community ontwikkelt ook de daadwerkelijke componenten en test ze in hun eigen websites en applicaties.
```

- **Voor leveranciers**: je kunt overheidsklanten makkelijker en consistenter bedienen.
- **Voor managers en beslissers**: je teams werken met bouwstenen die ook bij andere overheidsorganisaties worden bewezen.

## De NL Design System aanpak

### Onderdelen

NL Design System bevat verschillende soorten onderdelen:

> - [componenten](/componenten): bouwblokken waarmee websites kunnen worden opgebouwd, zoals knoppen, formuliervelden en alerts.

Author: aangepast! behalve bij eerste opsomming, waar ik ook code en ontwerp wilde laten terugkomen voor extra concreetheid

- [richtlijnen](/richtlijnen): adviezen over verschillende onderwerpen, zoals het [maken van formulieren](/richtlijnen/formulieren/).
- [patronen](/voorbeelden/patronen): verslagen van gebruikersonderzoek en inzichten

### Hoe we werken

> Deze onderdelen ontstaan meestal niet bij het kernteam, maar in de community. Bijvoorbeeld omdat een gemeente of ministerie een specifiek component nodig heeft. Als blijkt dat meer organisaties datzelfde componenten willen gebruiken, wordt er samengewerkt aan een ultieme versie van dat component, waarop het kernteam garanties geeft.

Author: Richtlijnen is de uitzondering waar ik aan dacht, maar dat is inderdaad wel echt een uitzondering

- **Samen gebouwd**: oplossingen worden gebouwd en gebruikt door een actieve community.​
- **Breed gedragen**: onderdelen zijn op meerdere plekken gebruikt en getest met zoveel mogelijk verschillende use cases.

## Toegankelijkheid

NL Design System helpt op dit moment op twee manieren bij de toegankelijkheidsopgave van organisaties:

- onze componenten kunnen worden gebruikt om toegankelijke producten te maken. Op “Hall of Fame” niveau garanderen we zelfs toegankelijkheid en zijn componenten in productie getest.
- onze richtlijnen kunnen worden gebruikt als kennisbank, ze bevatten concrete adviezen over wat er nodig is en waarom. Er zijn do's en don'ts, uitleg van WCAG criteria en tips die verder dan WCAG gaan.

## Leveranciers

Veel organisaties werken met NL Design System via hun leveranciers. Bijvoorbeeld doordat hun leveranciers onze componenten gebruiken, of meedoen in de community.

Ben je zelf leverancier en wil je voor klanten met NL Design System aan de slag? Neem contact op met het [kernteam](/project/kernteam), we helpen je graag op weg.

## Wie zijn jullie?

### Kernteam en community

> We zijn een samenwerking van specialisten vanuit de hele overheid (van gemeenten tot Rijk). Het [kernteam](/project/kernteam) faciliteert deze samenwerking. Specialisten zijn bijvoorbeeld ontwerpers, ontwikkelaars en onderzoekers. Ze werken in hun organisatie aan specifieke webprojecten werken en delen hun ervaring in de community.

Reviewer 2: ```suggestion
We zijn een samenwerking van specialisten binnen overheidsorganisaties. Het [kernteam](/project/kernteam) faciliteert deze samenwerking. Specialisten zijn bijvoorbeeld ontwerpers, ontwikkelaars en onderzoekers. Ze werken in hun organisatie aan specifieke projecten en delen hun kennis en ervaring binnen de community.
```

Author: Ik heb 'gemeenten' en 'Rijksoverheid' er wel in gelaten omdat veel mensen denken dat het alleen over gemeenten gaat

> We zijn een samenwerking van specialisten vanuit de hele overheid (van gemeenten tot Rijk). Het [kernteam](/project/kernteam) faciliteert deze samenwerking. Specialisten zijn bijvoorbeeld ontwerpers, ontwikkelaars en onderzoekers. Ze werken in hun organisatie aan specifieke webprojecten werken en delen hun ervaring in de community.

Author: dit van gemaakt: 

> Bij NL Design System werken specialisten uit overheidsorganisaties, van gemeenten tot Rijksoverheid, samen aan betere digitale dienstverlening.
